src/gantry/screens.py

```
@@ -109,30 +111,69 @@ def on_mount(self) -> None:
             pass
 
+    @work(thread=True)
+    def _load_namespaces_worker(self, context_name: str) -> None:
+        """Load available namespaces from the cluster in a background thread."""
+        namespaces = k8s.list_namespaces(context_name=context_name)
+        self.app.call_from_thread(self._set_namespaces, namespaces)
+
+    def _set_namespaces(self, namespaces: List[str]) -> None:
+        """Update the namespace list with loaded values."""
+        # Start with "all" option, then add actual namespaces
+        self.namespaces = ["all"] + sorted(namespaces)
```

Ruff (0.15.10)

[warning] Consider `["all", *sorted(namespaces)]` instead of concatenation

Replace with `["all", *sorted(namespaces)]`

(RUF005)